apps/meeting/scripts/update-react-library.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+/**
+ * This script clones and installs the latest code from amazon-chime-sdk-component-library-react
+ * GitHub repository when in meeting-dev branch.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { logger, spawnOrFail } = require('./utilities');
+const rimraf = require('rimraf');
+ 
+const currentBranch = spawnOrFail('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
+if (currentBranch === 'main') {
+  logger.log(`current branch is ${currentBranch}, using latest react library from NPM`);
```

Comment:
nit: `react library` => `Amazon Chime SDK React Components Library` to avoid confusion with Facebook's React library, or you can output the `package` variable at line 21.